checks/raw/branch_protection.go

```diff
@@ -66,6 +67,9 @@ func BranchProtection(cr *checker.CheckRequest) (checker.BranchProtectionsData,
 	// Get release branches.
 	releases, err := c.ListReleases()
 	if err != nil {
+		if errors.Is(err, clients.ErrUnsupportedFeature) {
+			return checker.BranchProtectionsData{}, nil
+		}
```

Comment:
we could still query branch protection of the default branch even if we can't determine the release branches. 
Would something like this be better?:
```go
// Get release branches.
releases, err := c.ListReleases()
if err != nil && !errors.Is(err, clients.ErrUnsupportedFeature) {
		return checker.BranchProtectionsData{}, fmt.Errorf("%w", err)
}
```